### PR TITLE
Make this addon FastBoot compatible

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
Tested on an actual app.

### CODE REVIEW

**Comment by kratiahuja** [comment]
Isn't this a dupe of #26?

**Comment by sivakumar-kailasam** [comment]
@kratiahuja No, apart from the build exclusion, I've handled couple of other places where there were errors in fastboot mode.

File: addon/helpers/is-clipboard-supported.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import Ember from 'ember';
-/* global Clipboard */
 
-export const isClipboardSupported = Clipboard.isSupported;
+export const isClipboardSupported = window.Clipboard ? window.Clipboard.isSupported: () => false;
```
Comment (jkusa) [inline addon/helpers/is-clipboard-supported.js]: Can we make this the following:

```js
window && window.Clipboard ? window.Clipboard.isSupported : () => false;
```
The docs suggest testing for the presence of `window`
https://ember-fastboot.com/docs/addon-author-guide#how-to-fix-it

**Comment by jkusa** [comment]
@sivakumar-kailasam thanks for this PR.  I have one comment. Otherwise, changes LGTM. After this PR lands, I will cut a release with an ember-cli upgrade and add fastboot as a dev dependency for the dummy app.

**Comment by sivakumar-kailasam** [comment]
Made the changes @jkusa. @kratiahuja out of curiosity in test mode, can we turn off fastboot mode via some config? Because tests might have different results for client side vs fastboot mode.

**Comment by kratiahuja** [comment]
Yeah you can turnoff fastboot via adding a query Param ?fastboot=false

**Pull request merged**